tests/cmdline/commands/test_export.py

```diff
@@ -213,6 +201,39 @@ def test_migrate_force(self):
                 self.assertTrue(os.path.isfile(filename_output))
                 self.assertEqual(zipfile.ZipFile(filename_output).testzip(), None)
 
+    def test_migrate_in_place(self):
+        """Test that passing the -i/--in-place option will overwrite the passed file."""
+        archive = 'export_v0.1_simple.aiida'
+        target_version = '0.2'
+        filename_input = get_archive_file(archive, filepath=self.fixture_archive)
+        filename_tmp = next(tempfile._get_candidate_names())  # pylint: disable=protected-access
+
+        try:
+            # copy file (don't want to overwrite test data)
+            shutil.copy(filename_input, filename_tmp)
+
+            # specifying both output and in-place should except
+            options = [filename_tmp, '--in-place', '--output-file', 'test.aiida']
+            result = self.cli_runner.invoke(cmd_export.migrate, options)
+            self.assertIsNotNone(result.exception)
```

Comment:
It's nice to print `stdout` if this fails, and vice versa when assert something in `stdout`, it's nice to print `stderr`.
If you find it confusing like this to read, you can also pass `result.output` in by using the `msg` parameter, i.e. `assertTrue(something, msg="this is shown if assertion fails")`.
```suggestion
            self.assertIsNotNone(result.exception, result.output)
```